packages/clay-badge/src/ClayBadge.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as React from 'react';
+import classNames from 'classnames';
+
+interface props {
+	label?: string | number;
+	className?: string;
```

Comment:
Maybe this should be `classNames`?

Reply:
I think it would confuse people to have `classNames` on components and then the native `className`(no "s") on elements. I think we either keep it consistent, or we entirely divert and use something like `elementClasses` that we had in metal.